packages/solidjs/package.json

```diff
@@ -54,6 +55,9 @@
     "solid-js": "1.8.11",
     "vite-plugin-solid": "^2.8.2"
   },
+  "optionalDependencies": {
+    "@solidjs/router": "0.13.3"
```

Comment:
Yeah, I'd do ^1.8.0 for `solid-js`. For the router it's harder because 🤷 damn 0.x versions 😬 at the very least we should also do `^0.13.3` but this means that for every minor release they do (e.g. 0.14.0) we'll have to update the SDK accordingly to match this 😬

Reply:
From the docs for the router:
> The docs are based on Solid Router v0.10.x. To use this version, you need to have Solid v1.8.4 or later installed.

Should I downgrade to that and see if it still runs?

Reply:
That puts us at `1.8.4` as minimum for solid-js. We can probably support a lower version if we have some kind of warnings inside `solidrouter.ts` if it's used with a version lower than `1.8.4`. Is that possible?

Reply:
IMHO it's fine to say this is `^1.8.4` too, this is greenfield so let's start with support that makes it reasonably easy for us! :)

Reply:
Updated to `^1.8.4`.